scripts/dot-createlicense.ps1

```diff
@@ -68,20 +60,20 @@ function CreateLicence {
 
             if ($Cloud -eq 'AWS') {
             # Write any old junk into the license file to obliterate the contents from the disk so it cannot be recovered
-            Get-Process | Out-File $licenseFile_
+            Get-Process | Out-File "a"
```

Comment:
The license file MUST be obliterated. If its only deleted it may be able to be recovered from disk. So, change `"a"` to `$licenseFile`